### PR TITLE
Add noop

### LINKED ISSUE #952: Add noop
op that does nothing - python script like `pass` so you can use it as a step to do custom things - navigate, publish..

### PR DESCRIPTION
**Pull request by Bonczidai**
fixes #952 

Signed-off-by: Bonczidai Levente <levente.bonczidai@hpe.com>

### CODE REVIEW

File: content/io/cloudslang/base/utils/noop.sl [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#   (c) Copyright 2015-2017 Hewlett-Packard Enterprise Development Company, L.P.
+#   All rights reserved. This program and the accompanying materials
+#   are made available under the terms of the Apache License v2.0 which accompany this distribution.
+#
+#   The Apache License is available at
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+########################################################################################################################
+#!!
+#! @description: noop should do "nothing". It simulates this behavior by calling the `pass` Python statement.
```
Comment (Bonczidai) [inline content/io/cloudslang/base/utils/noop.sl]: builder should fail if `@description` is missing - right now it doesn't - @tethryus WDYT?

Comment (tethryus) [inline content/io/cloudslang/base/utils/noop.sl]: @Bonczidai You are right. Nice catch.

Comment (Bonczidai) [inline content/io/cloudslang/base/utils/noop.sl]: opened an item for it: https://github.com/CloudSlang/cloud-slang/issues/985

**Comment by Bonczidai** [comment]
thx @tethryus for the review

**Pull request merged**